src/main/resources/application.properties

```diff
@@ -3,3 +3,8 @@ spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
 spring.datasource.url=jdbc:postgresql://${POLL_APP_DB_HOST}:5432/${POLL_APP_DATABASE}
 spring.datasource.username=${POLL_APP_ROOT_USERNAME}
 spring.datasource.password=${POLL_APP_ROOT_PASSWORD}
+
+
+poll.app.jwtSecret= bdqdgajkdnaidjwugebkfjjsfhisaudwiteaiksjlasfoioutoiyrqugdjdgfsfyfcusfuhjrgjobysuakdnfueiwufgwihfwyfdwuqiadggey
```

Comment:
Er ditta secretkey brukt for å opprette JWTs?

Reply:
Ja, hvis det finnes en automatisk måte å lage en secret på, kan jeg gjerne implementere den 😄

Reply:
Har noe på det, så vi kan ta det på møtet i morgen 😎